src/message_filters/__init__.py

```diff
@@ -43,9 +49,8 @@ def __init__(self):
     def registerCallback(self, cb, *args):
         """
         Register a callback function `cb` to be called when this filter
-        has output.
-        The filter calls the function ``cb`` with a filter-dependent list of arguments,
-        followed by the call-supplied arguments ``args``.
+        has output. The filter calls the function ``cb`` with a filter-dependent
```

Comment:
This looks like an unnecessary change of indention/whitespace only. We usually start new sentances on new lines in documentation to avoid reflowing whole paragraphs when making changes to wordings.